### DISCUSSION TITLE
Resource: jquery ui icons missing

### DISCUSSION THREAD

**Discussion by PiemP** [OrchardCMS/OrchardCore#6990] [General]
Hi all,

I have synchronized my branch with latest dev code and seems the jquery ui icons is missing.

![jquery ui missing icons](https://user-images.githubusercontent.com/2736009/91705520-5eac2200-eb7d-11ea-9038-aceb1baf6b16.jpg)

I have to open an issue about it or there is a reason about the absence of this file?

Thank you.

**Comment by Piedone** [comment]
Hmm, where is this file referenced?

**Reply by PiemP** [reply]
I use the JQuery UI date picker to render a text input that allow Italian date format in OC...is not "official", but since OC have it in the resources...before proceed with it I would get info about the management of this resource to understand if is an issue or if I need to do a customization of my admin theme to use a custom asset of JQuery UI with icons. 

I hope to have write something of understandable 😅

**Reply by Skrypt** [reply]
It's a regression made while making some cleanup in the OrchardCore.Resources module I think. The JQuery-ui files are not served from that module anymore. Though the JQuery-ui resource is still there and used in OC for some modules so this /Styles/images/ should be added back.

/cc @agriffard
